core/controllers/oppia_root.py

```diff
@@ -59,8 +59,11 @@ def get(self, **kwargs: Dict[str, str]) -> None:
         # The order of preference to determine the language direction is:
         # 1. Cookies
         # 2. Url params
-        # Incase we don't find a language direction from the above two, we
-        # default to AoT bundle.
+        # In the case we don't find a language direction from the above two,
+        # we default to AoT bundle.
+        # TODO(#16300): Refactor how we handle RTL CSS.
```

Comment:
Please explain this a bit more; "refactor" alone doesn't properly describe the desired end state. Imagine you're giving this TODO as an instruction to someone and they have to fix it based on what you've written -- it's important for them to at least know what the goal is.

Reply:
Done!